lib/graphql/batch.rb

```diff
@@ -19,6 +19,15 @@ def self.batch
       end
     end
 
+    def self.use(schema_defn)
+      if schema_defn.target.class != GraphQL::Schema
```

Comment:
I've just removed it. I don't think it's too helpful and it will have a fairly description error anyway.